WordPress/Classes/ViewRelated/Reader/ReaderPostCardCell.swift

```diff
@@ -231,6 +235,10 @@ fileprivate func < <T: Comparable>(lhs: T?, rhs: T?) -> Bool {
         WPStyleGuide.applyReaderSaveForLaterButtonStyle(saveForLaterButton)
     }
 
+    fileprivate func setupReblogActionButton() {
+        WPStyleGuide.applyReaderReblogActionButtonStyle(reblogActionButton)
```

Comment:
It looks like this method only sets the style. Maybe instead of a separate method, this line can just be added to `ApplyStyles`? I know this is legacy, but I think the same can be done for `setupSaveForLaterButton` (i.e. setting the styles in `ApplyStyles` and nixing `setupSaveForLaterButton` and `setupReblogActionButton`).

Reply:
Yeah they can and they should go in there, we reduce code lines and don't lose clarity